src/CoreFoundation/Dispatch.cs

```diff
@@ -234,7 +246,9 @@ public DispatchQueue (string label, Attributes attributes, DispatchQueue target
 			}
 		}
 
-#if !NET
+#if NET
+		[SupportedOSPlatform ("ios7.0")]
```

Comment:
Would this need a `[SupportedOSPlatform ("tvos")]` to show up as supported on tvOS as well? Or is that coming in a later pass on the attributs?